src/DatabaseHelper.ts

```diff
@@ -0,0 +1,139 @@
+import { Server } from 'http';
+import { Database, RunResult, Statement } from 'sqlite3';
+
+export default interface ServerInfo {
```

Comment:
```suggestion
export interface ServerInfo {
```
The `default` keyword is only used for when you import the file with `import SomeName from 'xy';`. You shouldn't be allowed to have one interface and one class both default exported at the same time. I wonder why TypeScript didn't complain